GiTinder/Models/Responses/ProfileResponse.cs

```diff
@@ -7,24 +7,31 @@
 namespace GiTinder.Models
 {
     public class ProfileResponse : GeneralApiResponseBody
+
     {
         [JsonProperty("username")]
         public string Username { get; set; }
-        [JsonProperty("url")]
-        public string Avatar { get; set; }
+
+        [JsonProperty("avatar_url")]
+        public string Url { get; set; }
+
         [JsonProperty("repos")]
         public string Repos { get; set; }
+
         [JsonProperty("languages")]
         public string Languages { get; set; }
 
+        [JsonProperty("snippets")]
+        public string Snippets { get; set; }
 
-        public ProfileResponse()
+        public ProfileResponse(string userName, string uRl, string rePos)
         {
-            Username = "aze";
-            Avatar = "https://avatars1.githubusercontent.com/u/5855091?s=400&v=4";
-            Repos = "string";
-            Languages = "eng";
-
+            Status = "ok";
+            Username = userName;
+            Url = uRl;
+            Repos = rePos;
+            Languages = "Java";
+            Snippets = "https://github.com/adamgyulavari/gf-chatapp";
```

Comment:
@dypher91 these 3 should be `list<string>`

Reply:
done